scripts/figure_setup/sync-data-files.R

```diff
@@ -65,7 +65,7 @@ sync_call <- glue::glue("aws s3 sync '{celltype_s3_files}' '{celltype_local_file
 system(sync_call)
 
 # sync merged object results for SCPCP000003 -----------------------------------
-merged_s3_dir <- "s3://nextflow-ccdl-results/scpca/processed/results/merged/SCPCP000003"
+merged_s3_dir <- "s3://nextflow-ccdl-results/scpca-prod/results/SCPCP000003/merged/"
```

Comment:
Let's not change this. This is a really big object and I was using the smaller one that had the specific libraries that we need to make this figure.

Reply:
I definitely get that, but `s3://nextflow-ccdl-results/scpca/processed/results/merged/SCPCP000003` doesn't exist anymore so this line doesn't do anything, and it won't copy down a smaller object.

```
(base) ~ ➤ op run -- aws s3 ls s3://nextflow-ccdl-results/scpca/processed/results/merged/SCPCP000003/                       
(base) ~ ➤                                                                                                   
```